### PR TITLE
Removes profileContacts toggle

### PR DESCRIPTION
**Pull request by radavis**
## Summary

- **Removes profileContacts toggle**

## Related issue(s)

- _Link to ticket created in va.gov-team repo_
department-of-veterans-affairs/va.gov-team#81337

## Testing done

- unit/e2e

## Screenshots

No UI changes

## What areas of the site does it impact?

*(Describe what parts of the site are impacted **if** code touched other areas)*

`/profile` and `/profile/contacts` paths

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### CODE REVIEW

**Review by adamwhitlock1 — APPROVED** [review]
this all looks good to me. Thanks for cleaning this up!

**Review by allisonlu — APPROVED** [review]
lgtm!!

File: src/applications/personalization/profile/tests/e2e/personal-health-care-contacts/contacts.cypress.spec.js [hunk 1]
```diff
@@ -21,8 +19,7 @@ describe('Personal health care contacts -- feature enabled', () => {
     ];
     mockGETEndpoints(otherEndpoints, 200, {});
 
-    featureToggles = generateFeatureToggles({ profileContacts: true });
-    cy.intercept('GET', '/v0/feature_toggles*', featureToggles);
+    cy.intercept('GET', '/v0/feature_toggles*', generateFeatureToggles());
```
Comment (carlosfelix2) [inline src/applications/personalization/profile/tests/e2e/personal-health-care-contacts/contacts.cypress.spec.js:22]: Remove "feature enabled" from test title
`describe('Personal health care contacts -- feature enabled',`

File: src/applications/personalization/profile/tests/e2e/profile-breadcrumbs.cypress.spec.js [hunk 2]
```diff
@@ -26,11 +26,7 @@ describe('Profile Breadcrumbs', () => {
       return;
     }
     it('render the active page name in the breadcrumbs', () => {
-      cy.intercept(
-        'GET',
-        '/v0/feature_toggles*',
-        generateFeatureToggles({ profileContacts: true }),
-      );
+      cy.intercept('GET', '/v0/feature_toggles*', generateFeatureToggles());
```
Comment (carlosfelix2) [inline src/applications/personalization/profile/tests/e2e/profile-breadcrumbs.cypress.spec.js:29]: Do you still need this line?

**Review by carlosfelix2 — APPROVED** [review]
Just a couple of comments

**Comment by carlosfelix2** [comment]
In the PR description, "What areas of the site does it impact?" is not filled in.

File: src/applications/personalization/profile/tests/e2e/profile-breadcrumbs.cypress.spec.js (hunk 2 above)
Comment (radavis) [reply src/applications/personalization/profile/tests/e2e/profile-breadcrumbs.cypress.spec.js:29]: Yep!

File: src/applications/personalization/profile/tests/e2e/personal-health-care-contacts/contacts.cypress.spec.js (hunk 1 above)
Comment (radavis) [reply src/applications/personalization/profile/tests/e2e/personal-health-care-contacts/contacts.cypress.spec.js:22]: I hope the authexp team will forgive me for this sin, as it feels silly to ask for a re-review for this text deletion.

**Review by asiisii — APPROVED** [review]
LGTM 🚀

**Review by jzucadi — APPROVED** [review]

**Review by rsmithadhoc — APPROVED** [review]

**Pull request merged**